.github/workflows/it.yaml

```diff
@@ -0,0 +1,99 @@
+name: Integration tests
+
+on:
+  push
+
+env:
+  KONG_VERSION: 2.8.1
+
+jobs:
+
+  prepare-and-run:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '17'
+          java-package: jdk
+
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '>=1.17.0'
+
+      - name: Build pep-plugin 
+        run: |
+          cd ./kong-pep-plugin 
+          docker run -v $(pwd):/temp/ --workdir="/temp" kong/go-plugin-tool:2.0.4-alpine-latest go build -ldflags "-X pep-plugin.Version=${{ needs.generate-version.outputs.version }}" -o pep-plugin .
+          cd ..
+          mkdir ./go-plugins/
+          cp ./kong-pep-plugin/pep-plugin ./go-plugins/pep-plugin
+
+
+      - name: Prepare values
+        run: | 
+          echo GITHUB_ORGANISATION: fiware >> values.yaml
+          echo GITHUB_REPO: kong-plugins-fiware >> values.yaml
+          echo VERSION: 0.0.1 >> values.yaml
+
+
+      - name: Parse rockspec
+        run: |
+          go install github.com/cbroglie/mustache/cmd/mustache@v1.3.1
+          ~/go/bin/mustache values.yaml kong-plugin-auth-endpoint-config-ishare/rockspec.mustache > kong-plugin-auth-endpoint-config-ishare/kong-plugin-auth-endpoint-config-ishare-0.0.1-1.rockspec
+          ~/go/bin/mustache values.yaml kong-plugin-ngsi-ishare-policies/rockspec.mustache > kong-plugin-ngsi-ishare-policies/kong-plugin-ngsi-ishare-policies-0.0.1-1.rockspec
+      
+      - uses: leafo/gh-actions-lua@v9
+      - uses: leafo/gh-actions-luarocks@v4
+      
+      - name: Make rock
+        run: |
+          cd kong-plugin-auth-endpoint-config-ishare
+          luarocks make
+          luarocks pack kong-plugin-auth-endpoint-config-ishare 0.0.1-1
+          cd ..
+          cd kong-plugin-ngsi-ishare-policies
+          luarocks make 
+          luarocks pack kong-plugin-ngsi-ishare-policies 0.0.1-1
+          cd ..
+          mkdir plugin-dir
+          cp -r kong-plugin-auth-endpoint-config-ishare/.  plugin-dir/
+          cp -r kong-plugin-ngsi-ishare-policies/.  plugin-dir/
+          cd plugin-dir/
+          luarocks install lua-fiware-lib
```

Comment:
Can't this be extracted somehow of the dependencies in the rock manifests?

Reply:
> Can't this be extracted somehow of the dependencies in the rock manifests?

It is in the release build, but there is no published version during the IT tests, thus I have to create them in-line(same as with pre-release).